README.md

```
@@ -68,6 +68,57 @@ Campaign states:
+```bash
+cd backend
+npm run mutation:ci
+```
+
+Outputs a compact text summary suitable for CI logs without generating the HTML report.
+
+### Mutation-killing tests
+
+Tests that specifically target surviving mutants live in:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)